src/transpiler/transpiler.ts

```diff
@@ -19,23 +19,24 @@ const ROOT_DIR = Path.resolve(__dirname, '../..');
 export async function transpileFiles(directory: string, outputDir: string, options?: TranspileFilesOptions) {
     const {files, dirs} = await getFilesInDir(directory);
     if(files.length > 0){
+        // source-map-support is required to make error stack traces work.
+        // Be carefull about what you add to the rollup process. 
+        // Any changes to the transpiled files might screw with the sourcemap linking to wrong code.
```

Comment:
A better explanation of this would be great. It seems a very dark comment, like "don't touch anything". It would be great if you could reference some docs or something illustrating your point here.

Reply:
Yea I see your point, i'll try to re-write it 🤔

Reply:
I am not able to directly find any information or references here. But I re-wrote it while including an example. Let me know what you think 😄

Reply:
Oh yeah! Much better now 😄